android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/NativeFormLangUtils.java

```diff
@@ -38,25 +59,25 @@ public static Context setAppLocale(Context ctx, String language) {
     }
 
     /**
-     * Performs translation on an interpolated {@param str}
+     * Performs translation on an interpolated {@param jsonFormString}
      * i.e. a String containing tokens in the format {{string_name}},
      * replacing these tokens with their corresponding values for the current Locale
      *
-     * @param str
+     * @param jsonFormString
      * @return
      */
-    public static String getTranslatedString(String str) {
-        String translationsFileName = getTranslationsFileName(str);
-        if (translationsFileName.isEmpty()) {
+    public static String getTranslatedString(String jsonFormString) {
```

Comment:
This method can translat any `String` and not just a `jsonFormString`, so this renaming is a bit misleading.